css-highlight-api-1/Overview.bs

```diff
@@ -309,12 +309,12 @@ Painting</h4>
 					<body>Lorem Ipsum.
 					<script>
 						let r1 = new Range();
-						r1.setStart(document.body, 1);
-						r1.setEnd(document.body, 5);
+						r1.setStart(document.body.childNodes[0], 1);
```

Comment:
I think it would be simpler to use `document.body.firstChild`.

Reply:
Makes sense, I'll change it